src/main/java/org/jabref/gui/fieldeditors/LinkedFileViewModel.java

```diff
@@ -107,7 +109,11 @@ public void acceptAsLinked() {
     }
 
     public Observable[] getObservables() {
-        return new Observable[] {this.downloadProgress, this.isAutomaticallyFound};
+        List<Observable> observables = new ArrayList<>(Arrays.asList(linkedFile.getObservables()));
+        observables.add(downloadOngoing);
+        observables.add(downloadProgress);
```

Comment:
also add `isAutomaticallyFound`

Reply:
Just budding in here... 

The file list would be cleaner if the filenames weren't shown when there is a description, as was done in JabRef 3.x.  If there are very long descriptions, it will still be cluttered, but in my opinion., tooltips aren't an improvement.  

The problem with descriptions only in tooltips is that you have to mouse over each one.  This makes JabRef harder to use:

1.  It forces you to lift you hands off the keyboard, but more importantly,
2. You can't scan the descriptions at a glance.  Instead, you have to mouse over the first one, then mouse over the next one (which causes the previous description to vanish), and so on.

You can simulate the user experience by having somebody else make you a JabRef entry with, say five files with slightly different descriptions -- this is what your own entries will look like to you, a couple years after you have made them.  You won't remember anything.

Now, try to decide which file to open.  With tooltips, you see only one description at a time, like you're viewing them through a straw.  Now compare that with the JabRef 3.x experience, where all descriptions are visible; with just one glance, you can comprehend all of them.

Reply:
Sounds reasonable!

Reply:
I now changed the order: Description is shown first and then the file link

Reply:
Come to think of it, you could even make the filename a tooltip.  Once you've got the description, the filename is probably not so useful, but some situation, you still might want to know it.

But then again, there is the "see everything at a glance" argument I just made...